src/PythonClient/src/quixstreams/dataframes/dataframe/dataframe.py

```diff
@@ -58,6 +57,18 @@ def process(self, row: Row) -> Optional[Union[Row, list[Row]]]:
         """
         return self._pipeline.process(row)
 
+    # TODO: maybe we should just allow list(Topics) as well (in many spots actually)
```

Comment:
Could you pls clarify this one? 
To me it's +- the same:
```
sdf.to_topic(topic1)
sdf.to_topic(topic2)

# vs 

sdf.to_topics([topic1, topic2])
```

Reply:
Resolving this as we discussed that we should just leave it as-is since it's not often you'd produce the exact same message to multiple topics.